web/src/user_profile.js

```diff
@@ -342,6 +361,18 @@ function render_bot_profile_fields(bot) {
     $("#avatar").css("background-image", `url(${people.medium_avatar_url_for_person(bot)})`);
 }
 
+export function render_user_profile_fields(user) {
```

Comment:
I think we can name this better by having something like `update_user_custom_profile_fields`.